sfcli/README.md

```diff
@@ -10,42 +10,45 @@ sfcli command [subcommand] [option1, option2, ...]
 ```
 
 ##### Example:
-Command "video" with subcommand "get" with no options
+Command "camera" with subcommand "get" with no options
 ```
- sfcli video get 
+ sfcli camera get 
 ```
 
-Will list all existing video processors. 
+Will list all existing camera processors. 
 
 ##### Options:
 Option are supported in long form prepended by double dash:
 ```
-sfcli video get --streamId 34
+sfcli camera get --streamId 2976ac85-f570-4bec-9561-ce5b2ec1d234
 ```
 as well in short form.
 ```
-sfcli video get -s:34
-sfcli video get -s=34
-sfcli video get -s 34
+sfcli camera get -s:2976ac85-f570-4bec-9561-ce5b2ec1d234
+sfcli camera get -s=2976ac85-f570-4bec-9561-ce5b2ec1d234
+sfcli camera get -s 2976ac85-f570-4bec-9561-ce5b2ec1d234
 ```
 
-### API destination
+### API location
 
 ##### HTTP
 
-Target HTTP API destination must defined by "--url" option. Value should be address of SmartFace HTTP endpoint with apropriate port.
+Target HTTP API location can be defined by "--host" option. Default value is "localhost".
 
 ```sh
-$ sfcli --url http://localhost:8099/
+$ sfcli --host localhost
```

Comment:
will this work with https?

Reply:
It won't. It is possible to have an option for protocol (http vs https) but thinking about it now, maybe we should have 2 url options (--odata-url and --rest-url) in case the APIs are not hosted on the same machine, or if they are hosted on some other URIs for example behind a reverse proxy

Reply:
but when you try to access remote server then you have to setup both options... 
what about we just add option --protocol with default value http?